CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @crosbymichael @cyphar @dqminh @giuseppe @hqhq @mrunalp @tianon @vbatts
```

Comment:
AFAIK this isn't *strictly speaking* necessary because we define the maintainers group with a GitHub group (and can require reviews -- this is what we do in runc) but I guess it doesn't hurt to put it in the repo as well.